### DISCUSSION TITLE
Form Wizard Error Message Translation

### DISCUSSION THREAD

**Discussion by webbipage** [filamentphp/filament#3263] [Help] [answered]
Question: Where to translate error message of required field of wizard layout?
Situation: Ignore required input field and click "Continue" of form wizard. The error messages appear with untranslated string.

**Comment by uadoronin** [comment, marked as answer]
If you run:
`php artisan vendor:publish --tag=filament-translations`
All filament language files will be in `lang/vendor/your-locale`.
Your validation error messages located at `lang/your-locale/validation.php`.

**Reply by webbipage** [reply]
Thank you for your answer.
